src/features/targeting/index.js

```diff
@@ -0,0 +1,4 @@
+// @flow
+export * from './contexts';
+
+export * from './types';
```

Comment:
Will this export the type without type keyword? When downcompiled, it won't export anything btw.

Reply:
it exports types correctly.